### ISSUE TITLE
Webpackv5 :: Invalid options object. Ignore Plugin has been initialized using an options object that does not match the API schema.

### ISSUE THREAD

**Issue by spnc-omz** [iamakulov/moment-locales-webpack-plugin#27]
Executing the MomentLocalesPlugin() in my project that i'm trying to test on Webpack V5 is throwing the following error:

``` Invalid options object. Ignore Plugin has been initialized using an options object that does not match the API schema.
 - options should be one of these:
   object { contextRegExp?, resourceRegExp? } | object { checkResource? }
ValidationError: Invalid options object. Ignore Plugin has been initialized using an options object that does not match the API schema.
    at validate (/Users/req/code/www/project.com/node_modules/webpack/node_modules/schema-utils/dist/validate.js:85:11)
    at new IgnorePlugin (/Users/req/code/www/project.com/node_modules/webpack/lib/IgnorePlugin.js:20:3)
    at new MomentLocalesPlugin (/Users/req/code/www/project.com/node_modules/moment-locales-webpack-plugin/index.js:118:16)
    at Object.<anonymous> (/Users/req/code/www/project.com/build-utils/webpack.common.js:112:9)
    at Module._compile (/Users/req/code/www/project.com/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (/Users/req/code/www/project.com/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/Users/req/code/www/project.com/build-utils/webpack.dev.js:6:22)
    at Module._compile (/Users/req/code/www/project.com/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (/Users/req/code/www/project.com/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at /Users/req/code/www/project.com/webpack.config.js:23:25
    at Array.forEach (<anonymous>)
    at module.exports (/Users/req/code/www/project.com/webpack.config.js:20:13)
    at handleFunction (/Users/req/code/www/project.com/node_modules/webpack-cli/bin/utils/prepareOptions.js:21:13)
    at prepareOptions (/Users/req/code/www/project.com/node_modules/webpack-cli/bin/utils/prepareOptions.js:9:5)
    at requireConfig (/Users/req/code/www/project.com/node_modules/webpack-cli/bin/utils/convert-argv.js:117:14)
    at /Users/req/code/www/project.com/node_modules/webpack-cli/bin/utils/convert-argv.js:123:17
    at Array.forEach (<anonymous>)
    at module.exports (/Users/req/code/www/project.com/node_modules/webpack-cli/bin/utils/convert-argv.js:121:15)
    at /Users/req/code/www/project.com/node_modules/webpack-cli/bin/cli.js:71:45
    at Object.parse (/Users/req/code/www/project.com/node_modules/webpack-cli/node_modules/yargs/yargs.js:567:18)
    at /Users/req/code/www/project.com/node_modules/webpack-cli/bin/cli.js:49:8
    at Object.<anonymous> (/Users/req/code/www/project.com/node_modules/webpack-cli/bin/cli.js:366:3)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18) 
```

Where this get's really silly, webpack updated their documentation on how to fix this issue, and it pretty cleanly translates to this package:

https://github.com/webpack/webpack.js.org/commit/9dc398b508752ddbf4516c31393b93ad2e6ff29c
https://webpack.js.org/plugins/ignore-plugin/

Occurs on https://github.com/iamakulov/moment-locales-webpack-plugin/blob/master/index.js#L118

**Comment by spnc-omz** [comment]
@iamakulov  if you have a contributors guide or workflow, I can try to help fix this, I just didn't see any and I don't know when I would be able to get to it. Let me know how I can help.

**Comment by iamakulov** [comment]
Thanks! The error was there because `MomentLocalesWebpackPlugin` used `IgnorePlugin` under the hood, and that plugin changed the API in webpack 5.

That’s now fixed in #30 and published in `1.2.0`! Now, the plugin is compatible with webpack 5 as well.

Additionally, I added contribution guidelines in #31.
